<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_rect_damper</class>
 <widget class="QDialog" name="Dialog_rect_damper">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>651</width>
    <height>561</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#Dialog_rect_damper
{
background-color:white;
}
QLineEdit 
{
font-family: 'Microsoft YaHei';
font-size: 13px;
}

QComboBox
{
font-family: 'Microsoft YaHei';
font-size: 13px;
}</string>
  </property>
  <widget class="QWidget" name="widget_top" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>651</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#widget_top
{
border: 1px solid #000000;
background-color: rgb(157, 196, 229);
	background-color: rgb(157, 196, 229);
}</string>
   </property>
   <widget class="QPushButton" name="close">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>5</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/images/image/closeWindow.jpg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_terminal" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>651</width>
     <height>301</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#widget_terminal
{
border: 1px solid #000000;
}</string>
   </property>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>651</width>
      <height>241</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#widget_2
{
border: 1px solid #000000;
}

QLabel
{
font-family: '黑体';
font-size: 16px;
}

QLineEdit
{
font-family: '黑体';
}

QRadioButton
{
font-family: '黑体';
}

QPushButton
{
font-family: '黑体';
}</string>
    </property>
    <widget class="QLineEdit" name="lineEdit_brand">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>40</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>40</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>-1</pointsize>
      </font>
     </property>
     <property name="text">
      <string>品牌</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>80</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>-1</pointsize>
      </font>
     </property>
     <property name="text">
      <string>阀门开度</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_angle">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>80</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>0°</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>15°</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>30°</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>45°</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>60°</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>75°</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>90°</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>80</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>-1</pointsize>
      </font>
     </property>
     <property name="text">
      <string>风量(m³/h)</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_air_volume">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>80</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget_3">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>120</y>
       <width>561</width>
       <height>72</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_10">
        <item>
         <widget class="QLabel" name="label_23">
          <property name="minimumSize">
           <size>
            <width>51</width>
            <height>31</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>51</width>
            <height>31</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>黑体</family>
            <pointsize>-1</pointsize>
           </font>
          </property>
          <property name="text">
           <string>63Hz</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_63">
          <property name="minimumSize">
           <size>
            <width>51</width>
            <height>31</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>51</width>
            <height>31</height>
           </size>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_11">
        <item>
         <widget class="QLabel" name="label_24">
          <property name="minimumSize">
           <size>
            <width>51</width>
            <height>31</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>51</width>
            <height>31</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>黑体</family>
            <pointsize>-1</pointsize>
           </font>
          </property>
          <property name="text">
           <string>125Hz</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_125">
          <property name="minimumSize">
           <size>
            <width>51</width>
            <height>31</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>51</width>
            <height>31</height>
           </size>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_12">
        <item>
         <widget class="QLabel" name="label_25">
          <property name="minimumSize">
           <size>
            <width>51</width>
            <height>31</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>51</width>
            <height>31</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>黑体</family>
            <pointsize>-1</pointsize>
           </font>
          </property>
          <property name="text">
           <string>250Hz</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_250">
          <property name="minimumSize">
           <size>
            <width>51</width>
            <height>31</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>51</width>
            <height>31</height>
           </size>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_13">
        <item>
         <widget class="QLabel" name="label_26">
          <property name="minimumSize">
           <size>
            <width>51</width>
            <height>31</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>51</width>
            <height>31</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>黑体</family>
            <pointsize>-1</pointsize>
           </font>
          </property>
          <property name="text">
           <string>500Hz</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_500">
          <property name="minimumSize">
           <size>
            <width>51</width>
            <height>31</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>51</width>
            <height>31</height>
           </size>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_14">
        <item>
         <widget class="QLabel" name="label_27">
          <property name="minimumSize">
           <size>
            <width>51</width>
            <height>31</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>51</width>
            <height>31</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>黑体</family>
            <pointsize>-1</pointsize>
           </font>
          </property>
          <property name="text">
           <string>1kHz</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_1k">
          <property name="minimumSize">
           <size>
            <width>51</width>
            <height>31</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>51</width>
            <height>31</height>
           </size>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_15">
        <property name="spacing">
         <number>6</number>
        </property>
        <item>
         <widget class="QLabel" name="label_28">
          <property name="minimumSize">
           <size>
            <width>51</width>
            <height>31</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>51</width>
            <height>31</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>黑体</family>
            <pointsize>-1</pointsize>
           </font>
          </property>
          <property name="text">
           <string>2kHz</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_2k">
          <property name="minimumSize">
           <size>
            <width>51</width>
            <height>31</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>51</width>
            <height>31</height>
           </size>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_16">
        <item>
         <widget class="QLabel" name="label_1">
          <property name="minimumSize">
           <size>
            <width>51</width>
            <height>31</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>51</width>
            <height>31</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>黑体</family>
            <pointsize>-1</pointsize>
           </font>
          </property>
          <property name="text">
           <string>4kHz</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_4k">
          <property name="minimumSize">
           <size>
            <width>51</width>
            <height>31</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>51</width>
            <height>31</height>
           </size>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_17">
        <item>
         <widget class="QLabel" name="label_29">
          <property name="minimumSize">
           <size>
            <width>51</width>
            <height>31</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>51</width>
            <height>31</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>黑体</family>
            <pointsize>-1</pointsize>
           </font>
          </property>
          <property name="text">
           <string>8kHz</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_8k">
          <property name="minimumSize">
           <size>
            <width>51</width>
            <height>31</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>51</width>
            <height>31</height>
           </size>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_18">
        <item>
         <widget class="QLabel" name="label_30">
          <property name="minimumSize">
           <size>
            <width>77</width>
            <height>31</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>51</width>
            <height>31</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>黑体</family>
            <pointsize>-1</pointsize>
           </font>
          </property>
          <property name="text">
           <string>总值dB(A)</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_total">
          <property name="minimumSize">
           <size>
            <width>77</width>
            <height>31</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>51</width>
            <height>31</height>
           </size>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>40</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>-1</pointsize>
      </font>
     </property>
     <property name="text">
      <string>型号</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_model">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>40</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_confirm">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>200</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
	color:#ffffff; /*文字颜色*/
    background-color: #00ACB9;/*背景色*/
    border-radius:0px; /*边框倒角*/
	border: 1px solid #000000;
}


QPushButton:hover 
{
background-color: rgb(0,142,153);
color: white;
}</string>
     </property>
     <property name="text">
      <string>确认</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>-1</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid #000000;</string>
     </property>
     <property name="text">
      <string>厂家信息</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_data_source">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>40</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>厂家信息</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>设计预估</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>40</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>-1</pointsize>
      </font>
     </property>
     <property name="text">
      <string>来源</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_18">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>80</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>-1</pointsize>
      </font>
     </property>
     <property name="text">
      <string>长(mm)</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_length">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>80</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_width">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>80</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_19">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>80</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>-1</pointsize>
      </font>
     </property>
     <property name="text">
      <string>宽(mm)</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget_7">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>10</y>
      <width>228</width>
      <height>54</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_20">
     <item>
      <widget class="QLabel" name="label_20">
       <property name="minimumSize">
        <size>
         <width>20</width>
         <height>41</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>20</width>
         <height>41</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel { 
border: none; 
background-color : rgb(105, 105, 105);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_21">
       <property name="font">
        <font>
         <family>Microsoft YaHei</family>
         <pointsize>24</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel { 
border: none; 
color : rgb(105, 105, 105);
}</string>
       </property>
       <property name="text">
        <string>方形调风门</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignBottom|Qt::AlignHCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>330</y>
     <width>651</width>
     <height>231</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid #000000;</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>40</y>
      <width>521</width>
      <height>161</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/images/image/damper_desci.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>经验公式</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEdit_model</tabstop>
  <tabstop>lineEdit_brand</tabstop>
  <tabstop>lineEdit_length</tabstop>
  <tabstop>lineEdit_width</tabstop>
  <tabstop>lineEdit_air_volume</tabstop>
  <tabstop>lineEdit_63</tabstop>
  <tabstop>lineEdit_125</tabstop>
  <tabstop>lineEdit_250</tabstop>
  <tabstop>lineEdit_500</tabstop>
  <tabstop>lineEdit_1k</tabstop>
  <tabstop>lineEdit_2k</tabstop>
  <tabstop>lineEdit_4k</tabstop>
  <tabstop>lineEdit_8k</tabstop>
  <tabstop>pushButton_confirm</tabstop>
  <tabstop>lineEdit_total</tabstop>
  <tabstop>comboBox_angle</tabstop>
  <tabstop>close</tabstop>
  <tabstop>comboBox_data_source</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
